package.json

```diff
@@ -29,7 +29,7 @@
     "@types/react-redux": "7.1.9",
     "@types/react-router-dom": "5.1.5",
     "@types/toastr": "2.1.38",
-    "axios": "0.20.0",
+    "axios": "^0.21.1",
```

Comment:
Use exact versions of npm packages

https://medium.com/front-end-weekly/how-to-save-exact-npm-package-versions-7f8ff9a0415a

Reply:
added .npmrc with save-exact=true to the project

Reply:
and changed package.json